SampleScripts/PowerManagement-ManagementDomain.ps1

```diff
@@ -363,15 +364,17 @@ if ($PsBoundParameters.ContainsKey("shutdown") -or $PsBoundParameters.ContainsKe
                     Write-PowerManagementLogMessage -Type INFO -Message "A single cluster exists in the management domain."
             }
     
-
+            $cluster = Get-VCFCluster | Where-Object { $_.id -eq ($workloadDomain.clusters.id) }
```

Comment:
$cluster = Get-VCFCluster | Where-Object { $_.id -eq ($workloadDomain.clusters.id) }

should be within the else{} for when there's a single cluster exists in the management domain.